frontend/src/metabase/components/DebouncedFrame.jsx

```diff
@@ -32,28 +32,34 @@ export default class DebouncedFrame extends React.Component {
     };
   }
 
-  setSize = (width, height) => {
+  updateSize = () => {
     this._transition = false;
-    this.setState({ width, height }, this._updateTransitionStyle);
+
+    const { width, height } = this.props;
+    if (width !== this.state.width || height !== this.state.height) {
+      this.setState({ width, height }, this._updateTransitionStyle);
+    } else {
+      this._updateTransitionStyle();
+    }
   };
 
-  setSizeDebounced = _.debounce(this.setSize, DEBOUNCE_PERIOD);
+  updateSizeDebounced = _.debounce(this.updateSize, DEBOUNCE_PERIOD);
 
   UNSAFE_componentWillReceiveProps(nextProps) {
     if (!nextProps.enabled) {
       this._updateTransitionStyle();
       return;
     }
     if (
-      this.props.width !== nextProps.width ||
-      this.props.height !== nextProps.height
+      this.state.width !== nextProps.width ||
+      this.state.height !== nextProps.height
     ) {
       if (this.state.width == null || this.state.height == null) {
-        this.setSize(nextProps.width, nextProps.height);
+        this.updateSizeDebounced();
```

Comment:
Shouldn't it be `this.setSizeDebounced(nextProps.width, nextProps.height);` ?
Merge conflict or am I misreading the two PRs?

Reply:
@flamber The first PR [change the line to](https://github.com/metabase/metabase/pull/20954/files#diff-cfcbb156d18ef93cd9c3a11763ba671ebe6c937102ecd10412113b745362eb7aR52) `this.setSizeDebounced(nextProps.width, nextProps.height);` then the [second PR which fixes the regression change it to](https://github.com/metabase/metabase/pull/21051/files#diff-cfcbb156d18ef93cd9c3a11763ba671ebe6c937102ecd10412113b745362eb7aR58) `this.updateSizeDebounced()`.

This is already correct.

Reply:
Sorry, got lost in the commits